drone_dji_sdk.orogen

```diff
@@ -32,7 +32,7 @@ task_context "DroneFlightControlTask" do
     runtime_states :TELEMETRY, :CONTROLLING, :CONTROL_LOST
 
     # Deployed with a 10ms period by default
```

Comment:
Need to update the comment. Use it to explain the rationale for such a low period.